fluidplayer.js

```diff
@@ -697,18 +697,65 @@ var fluidPlayerClass = {
         var player = this;
         var videoPlayerTag = document.getElementById(this.videoPlayerId);
         var list = [];
-        list.length = 0;
+        var vastList = Object.values(player.adList);
+        
+        if (roll == 'preRoll') {
+            for (var [key, vast] of vastList.entries()) {
```

Comment:
This construct is not supported in IE, even latest.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Browser_compatibility
I know the usage of it is dying, but it ain't dead yet.